docker/1.4.4/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine:3.3
+MAINTAINER David Audet <david.audet@ca.com>
+
+LABEL Description="Mosquitto MQTT Broker"
```

Comment:
Why? In the long form description, sure, but why should it be in the top level label?